src/config_class.py

```diff
@@ -58,6 +71,7 @@ class BaldOrNotConfig:
     training_params: TrainingParams = field(
         default_factory=lambda: TrainingParams()
     )
+    tuning_params: TuningParams = field(default_factory=lambda: TuningParams())
```

Comment:
use abstract config and TrainingConfig and TuningConfig

Reply:
ok